### PR TITLE
Yield timeout should be common to all loaders as framerate is an appl…

### PR DESCRIPTION
**Pull request by botrif**
…ication-wide concern

### CODE REVIEW

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs [hunk 1]
```diff
@@ -139,7 +136,6 @@ protected struct GLBStream
 			public long StartPosition;
 		}
 
-		protected float _timeAtLastYield = 0f;
 		protected AsyncCoroutineHelper _asyncCoroutineHelper;
```
Comment (blgrossMS) [inline UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs]: Shouldn't GLTFSceneImporter now take an IAsyncCoroutineHelper?

File: UnityGLTF/Assets/UnityGLTF/Scripts/Async/AsyncCoroutineHelper.cs [hunk 2]
```diff
@@ -4,18 +4,22 @@
...
+		public float BudgetPerFrameInSeconds = 0.01f;
+
 		private Queue<CoroutineInfo> _actions = new Queue<CoroutineInfo>();
+		private WaitForEndOfFrame _waitForEndOfFrame = new WaitForEndOfFrame();
+		private float _timeout;
 
 		public Task RunAsTask(IEnumerator coroutine, string name)
 		{
-			TaskCompletionSource<bool> tcs = new TaskCompletionSource<bool>();
+			var tcs = new TaskCompletionSource<bool>();
```
Comment (blgrossMS) [inline UnityGLTF/Assets/UnityGLTF/Scripts/Async/AsyncCoroutineHelper.cs]: nit- better to have type here

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs (hunk 1 above)
Comment (botrif) [reply UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs]: Will do

File: UnityGLTF/Assets/UnityGLTF/Scripts/Async/AsyncCoroutineHelper.cs (hunk 2 above)
Comment (botrif) [reply UnityGLTF/Assets/UnityGLTF/Scripts/Async/AsyncCoroutineHelper.cs]: OK

**Pull request merged**